Dockerfile

```diff
@@ -29,3 +31,7 @@ ENTRYPOINT ~/.local/share/pypoetry/venv/bin/poetry run flask run --host 0.0.0.0
 FROM base as tests
 
 ENTRYPOINT ~/.local/share/pypoetry/venv/bin/poetry run pytest tests
+
+FROM framework as dependency-scan
+
+ENTRYPOINT ~/.local/share/pypoetry/venv/bin/poetry run safety check
```

Comment:
Would it be worth updating the PATH so that we can reduce these long commands? E.g. adding the below before the `poetry install` step
```
ENV PATH=/root/.local/share/pypoetry/venv/bin:${PATH}
```